mkdocs.yml

```diff
@@ -51,14 +51,16 @@ theme:
     text: Inter
   palette:
     - media: "(prefers-color-scheme: light)"
-      scheme: default  # TODO: custom kolena-light theme
+      scheme: default
+      accent: cyan
       toggle:
-        icon: material/toggle-switch
+        icon: material/lightbulb
```

Comment:
Non-blocking preference discussion: personally lightbulb is a little un-intuitive to me. I've seen sun/moon icon, what do others think of that if they are available?

Reply:
Updated to use sun/moon iconography.